controller/src/main/java/org/jboss/as/controller/persistence/yaml/YamlConfigurationExtension.java

```diff
@@ -597,12 +598,22 @@ private class OperationConstructor extends Constructor {
         private final Tag UNDEFINE = new Tag("!undefine");
         private final Tag ADD = new Tag("!list-add");
 
-        public OperationConstructor() {
+        public OperationConstructor(LoaderOptions loadingConfig) {
+            super(loadingConfig);
             this.yamlConstructors.put(REMOVE, new ConstructRemoveOperation());
             this.yamlConstructors.put(UNDEFINE, new ConstructUndefineOperation());
             this.yamlConstructors.put(ADD, new ConstructListAddOperation());
         }
 
+        @Override
+        protected Class<?> getClassForNode(Node node) {
+            Class<? extends Object> classForTag = typeTags.get(node.getTag());
+            if (classForTag == null) {
+                throw new YAMLException("Class not found: " + node.getTag().getClassName());
```

Comment:
This is OK to go without i18n. The parent class uses the same message and it is not internationalized.